tox.ini

```diff
@@ -11,11 +11,12 @@ minversion = 4.2.4
 requires = virtualenv>=20.16.6
 
 [testenv]
-extras = yara
+extras = full
```

Comment:
Wouldn't it be nicer to change the dependency in pyproject.toml? I guess most people who want to use the yara functionality don't want the hassle of compiling the yara-python package?